.github/workflows/cli_release.yml

```diff
@@ -42,6 +42,21 @@ jobs:
         with:
           workspaces: packages/cli -> ../../target
 
+      # Replace the tag for the cargo-binstall record with the current release tag
+      - name: Replace pkg-url in Cargo.toml and push changes to tag # Only run on Linux
+        if: ${{matrix.platform.target == 'x86_64-unknown-linux-gnu'}}
+        run: |
+          # Replace the tag for the cargo-binstall record with the current release tag
+          sed -i 's/\/{ tagName }\//\/${{ github.event.release.tag_name }}\//g' packages/cli/Cargo.toml
+
+          # Commit and push the changes to the tag
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com" # This is a dummy email, copied from https://github.com/stefanzweifel/git-auto-commit-action
+          git config --local user.name "github-actions[bot]"
+          git add packages/cli/Cargo.toml
+          git commit -m "Update pkg-url to ${{ github.event.release.tag_name }}"
+          git tag -fa ${{ github.event.release.tag_name }} -m "Update pkg-url to ${{ github.event.release.tag_name }}"
+          git push origin --tags --force
```

Comment:
It looks like you update the tag that caused the CI to run and override the description. Is there a reason we need to override the message for cargo-bininstall. If we don't do that, can we avoid force pushing to the main branch. It might be fine, but I'm a bit worried about force pushing in CI

Reply:
So we need to pass it some tag description in order to generate the tag. If the release notes are generated in GitHub, they actually aren't added to the tag annotations. I think this is an optimization to keep the .git index smaller. GitHub will only show the description from the tag annotations if the release body is empty. I have tested to make sure that the release body is not overwritten when the tag is force pushed. I could copy the contents of the tag annotation into the new tag, it will just likely be empty if the tag is created in GitHub, as opposed to in the git client.

Reply:
I am hesitant to copy the contents of the release body into the tag description, though, as that could bloat the repo index.

Reply:
If I use a lightweight tag I can avoid the message altogether, but then there's no record of who created the tag

Reply:
I think the empty tag github creates automatically is ok if we can avoid force pushing. If we want a more detailed git tag we can create that in the CLI before we publish a release. Or CI could create it when we create a workflow for creating releases

Reply:
I've changed the tag so it now creates a lightweight tag without a description. This will still overwrite any tag annotations that were added to the previous tag. That means that release notes must be written via the GitHub release instead of in the tag message, but from what I can tell, all recent releases already do this so it shouldn't be a problem.

Reply:
Is the force push still necessary with the lightweight tag?

Reply:
Yes, the force push is still necessary as we are replacing an existing tag in the remote. Without the force push, we get the following error:
```txt
To https://github.com/esimkowitz/dioxus
 ! [rejected]        CLI-v0.4.1.20 -> CLI-v0.4.1.20 (already exists)
error: failed to push some refs to 'https://github.com/esimkowitz/dioxus'
hint: Updates were rejected because the tag already exists in the remote.
```